src/_data/how_to/get_connected.yml

```diff
@@ -52,30 +59,14 @@ toc:
   - title: New data plan options
     subsection_items:
-      - title: What is FirstNet? How is it different from other broadband or emergency responder networks?
-        url: "what-is-firstnet"
-        paragraphs:
-          - text: The FirstNet network was created and is sustained through a contract with the Federal Government. It is designed to offer a dedicated network for approved first responders. In California, transit providers are considered first responders and can access the FirstNet network.
-          - text: FirstNet is different from other cellular networks in that it provides priority for first responders automatically. This limits the potential for you to lose critical service capabilities during demanding times. In comparison to local emergency responder networks, FirstNet provides broadband voice, data, and video capabilities whereas typical emergency responder networks provide voice services.
       - title: Why do the FirstNet plans have tiers? What do these tiers mean?
         url: "firstnet-tiers"
         paragraphs:
-          - text: Cal-ITP worked with FirstNet to create data plans tailored for transit agencies at a competitive price. The tiers incentivize more collective purchases and were designed by FirstNet to enable discounted rates in a way that works for their business case.
+          - text: FirstNet created data plans tailored for transit agencies at a competitive price. The tiers incentivize more collective purchases and were designed to enable discounted rates in a way that works for their business case.
           - text: The FirstNet tier prices are based on bulk volume purchased on that contract vehicle, (i.e., total number purchased by all agencies combined, not the number of SIMs purchased by an individual agency). Tier II pricing is guaranteed through March 2024. When the Tier III threshold is reached, all agencies will see their plans further discounted.
       - title: Can FirstNet be used for any device and use case?
         url: "firstnet-devices-use-case"
         paragraphs:
-          - text: FirstNet only works for operations-facing use cases, given it is a first responder network. This means it cannot be used for passenger WiFi. Additionally, devices are required to have an additional security certification to operate on the federal network. This certification is common but not universal and should be confirmed before purchase.
+          - text: FirstNet only works for operations-facing use cases, given it is a first responder network. This means it cannot be used for passenger WiFi. Additionally, devices are required to have an additional security certification to operate on the federal network. This certification is common but not universal. For agencies purchasing hardware devices off of MobiMart, the table below summarizes what networks each device is certified to operate on.
```

Comment:
@mrose914 

Is `MobiMart` an acceptable way to refer to Mobility Marketplace in user-facing copy like this? Isn't it more of an internal nickname?